src/lgdo/types/table.py

```diff
@@ -140,8 +140,19 @@ def add_field(
 
         super().add_field(name, obj)
 
+        if not self.size:
```

Comment:
This should be:
```suggestion
        if self.size is not None:
```

Reply:
Implemented it as `if self.size is None:` (w/o the `not`)

Reply:
yes sorry